openedx/features/course_experience/templates/course_experience/mobile/course-dates-fragment.html

```diff
@@ -1,9 +1,15 @@
 ## mako
 
 <%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
```

Comment:
Is this necessary? It doesn't look like you are translating anything in this file in your changes? Is it for the include in the case that's included? Is it possible to just move this import into that file?